user/util.go

```diff
@@ -1,17 +1,20 @@
 package user
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
 	neofscrypto "github.com/nspcc-dev/neofs-sdk-go/crypto"
 )
 
+var ErrOwnerExtract = errors.New("decode owner failed")
```

Comment:
Exported thing -> there has to be a description for it.

Reply:
I see it in the subsequent commit, but again, better have it in the same one.

Reply:
Agree, I'm working on it - having docs in the same commit